airbyte-integrations/connectors/source-file/source_file/client.py

```diff
@@ -19,6 +19,7 @@
 from genson import SchemaBuilder
 from google.cloud.storage import Client as GCSClient
 from google.oauth2 import service_account
+from yaml import safe_load
```

Comment:
`yaml` is included via cdk right?

Reply:
Not sure where it's coming from exactly but i didn't need to add it to any `setup.py` or `requirements.txt`, seems to be standard now though?